src/m365/spo/commands/contenttype/contenttype-field-list.ts

```diff
@@ -0,0 +1,174 @@
+import { Logger } from '../../../../cli/Logger.js';
+import GlobalOptions from '../../../../GlobalOptions.js';
+import { formatting } from '../../../../utils/formatting.js';
+import { odata } from '../../../../utils/odata.js';
+import { urlUtil } from '../../../../utils/urlUtil.js';
+import { validation } from '../../../../utils/validation.js';
+import SpoCommand from '../../../base/SpoCommand.js';
+import commands from '../../commands.js';
+
+interface CommandArgs {
+  options: Options;
+}
+
+export interface Options extends GlobalOptions {
+  webUrl: string;
+  contentTypeId?: string;
+  contentTypeName?: string;
+  listTitle?: string;
+  listId?: string;
+  listUrl?: string;
+  properties?: string;
+}
+
+class SpoContentTypeFieldListCommand extends SpoCommand {
+  public get name(): string {
+    return commands.CONTENTTYPE_FIELD_LIST;
+  }
+
+  public get description(): string {
+    return 'Lists fields for a given site or list content type';
+  }
+
+  public defaultProperties(): string[] | undefined {
+    return ['Id', 'Title', 'Group', 'Hidden'];
```

Comment:
I would suggest we swap out the `Group` property for `InternalName` as that is a more interesting value to have for a field.

Reply:
Consistency wise, should I also change this in spo field list?

Reply:
That seems like a good idea!